submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+Your other big concern is fluid flow, and the main constraint is the heat flow along heat pipes. The farther away the heat gets from the reactors, the lower the maximum temperature the pipes can have. Since reactors temperatures cannot exceed 1000C, heat pipes beyond a certain distance become too cold to operate heat exchangers. As the [wiki shows](https://wiki.factorio.com/Tutorial:Nuclear_power#Heat_pipes) and [this post effectively visualizes](https://www.reddit.com/r/factorio/comments/ge4y6c/heat_pipe_throughput_and_a_bonus_note_on_parallel/),  a single pipe can heat a row of at most 21 heat exchangers, but you can improve the range here by making the pipe thicker and by adding heat exchangers on both sides of it instead of one side.  Another tricked used by the community here is placing unfueled nuclear reactors to act as 5x5 heat pipes because this allows travelling the same distance with fewer entities and connecting to more heat exchangers from a single entity. In any case, your best bet is to keep your heat pipes short by keeping heat exchangers nearby. 
+
+As for the other fluids, you want to ensure that the water from your offshore pumps is reaching your heat exhangers without significant [throughput loss in pipelines](https://wiki.factorio.com/Fluid_system#Pipelines), hence make sure to have multiple pipelines along the way and regularly place pumps on them. Secondly, you want to make sure the steam generated has enough pipe space to flow from exchangers to turbines without backing up, which is usually not a big issue and can be solved again using more pipes and pumps.
+
+### What about storing the steam?
+
+There is some debate in the community about whether steam storage is worthwhile. It is worth noting that it is not necessary for normal plant operation, since a correct ratio of exchangers to turbines (with *just* more than enough turbines) would ensure any steam produced is being consumed while the plant is running at full capacity. Meanwhile, adding optional components for steam storage, especially at a large scale, inflicts a toll on UPS. This is the main argument against steam storage and it is a valid one, but the practise it is not without its niche uses.

```

Comment:
you say the toll on UPS is valid, without demonstrating/advancing arguments for this?

Reply:
Right, I will cite u/warbaque's analysis!